src/app/components/Resume/ResumePDF/common/index.tsx

```diff
@@ -96,7 +96,7 @@ export const ResumePDFBulletList = ({
               }}
               bold={true}
             >
-              {"•"}
+              {item !== "" ? "•" : ""}
```

Comment:
```suggestion
              { item ?  "•" : ""}
```
what if we check for all `falsy` values :thinking:

Reply:
Good call, thanks!